lib/screens/music_screen.dart

```diff
@@ -260,6 +265,13 @@ class _MusicScreenState extends ConsumerState<MusicScreen>
                       )
                     ]
                   : [
+                      if (!Platform.isIOS && !Platform.isAndroid)
```

Comment:
The platform filter is just there because there isn't enough space, correct?  
If so, is there a way to use this refresh behavior also for the swiping down to refresh? Currently the only way to hard refresh is to toggle offline mode...

Reply:
The swiping down, any settings/filter changes, and this button do the same refresh.  I messed with it to harden the refresh when I unified the online and offline paging during the downloads rework, so the pull refresh should already be just as good as the offline toggle.

Reply:
Okay, maybe I'm just still used to the old behavior and didn't "trust" the refresh enough. Now that you mention it, every time I did use the refresh, it actually did load correctly, so I guess it works fine :)